package.json

```diff
@@ -652,6 +681,12 @@
           "command": "zowe.uss.createNode",
           "group": "inline"
         },
+        {
+
+          "when": "viewItem =~ /^ussSession.*/",
```

Comment:
This regExp will be true for ussSession and favorite ussSession too. Was that the intention or did yo also want to exclude the favorites? `viewItem =~ /^(?!.*_fav.*)ussSession.*/`

Reply:
meaning the edit icon will also appear in favorites? Somehow this capability doesn't makes sense to me.. hehe. I'm slow, i guess. In my POV, the edit icon should be similar to the Create Dataset / Filter Dataset Icon which doesn't appear in favorites. That is why I patterned it under inline group. Does this makes sense?

Reply:
Datasets was a good example to choose because it has the (?!.*_fav.*) negative lookup at the front. What this means is that a check is made to see if the item is a favorite before testing to see if it was a session. If it is found to be a favorite it fails straight away.

ussSession is wrong and probably my fault.  
```json
        {
          "when": "viewItem =~ /^ussSession.*/",
          "command": "zowe.uss.fullPath",
          "group": "inline"
        },
        {
          "when": "viewItem =~ /^ussSession.*/",
          "command": "zowe.uss.createNode",
          "group": "inline"
        },
        {
          "when": "viewItem =~ /^ussSession.*/",
          "command": "zowe.uss.editSession",
          "group": "inline"
        },
```
If these were a favorite search you have the search, create and edit inline menus so they should all be 
```json
        {
          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
          "command": "zowe.uss.fullPath",
          "group": "inline"
        },
        {
          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
          "command": "zowe.uss.createNode",
          "group": "inline"
        },
        {
          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
          "command": "zowe.uss.editSession",
          "group": "inline"
        },
```

Reply:
a ok. I can change that. I'll do it now.

Reply:
Thank you, my mistake leaving the ussSession like that